credentials/tls/certprovider/pemfile/watcher_test.go

```diff
@@ -55,6 +56,29 @@ func Test(t *testing.T) {
 	grpctest.RunSubTests(t, s{})
 }
 
+func compareKeyMaterial(got, want *certprovider.KeyMaterial) error {
+	// TODO(easwars): Remove all references to reflect.DeepEqual and use
+	// cmp.Equal instead. Currently, the later panics because x509.Certificate
```

Comment:
Nit: "latter"

Reply:
Changed the comment since I have switched to `cmp.Equal()` with a `nil` check.